R/network_structure.R

```diff
@@ -45,26 +45,26 @@ network.structure <- function(data.nma, my_order=NA) {     # data.nma is a BUGSn
   pairs2 <- data.frame(from = pairs[1,],
```

Comment:
There's a new R pipe |> (I know this is just edits for scoping but worth incorporating where possible)

Reply:
This new pipe was introduced in 4.0+, If we choose to included the new pipe we will need to state that this app is only compatible with 4.0+ if running locally. I know some people are still running 3.6.3 etc.